### DISCUSSION TITLE
[Question] Build app that uses SQLAlchemy

### DISCUSSION THREAD

**Discussion by MoreiraCristiano** [flet-dev/flet#2769] [Q&A]
### Question

How can I build an app that uses the SQLAlchemy orm? I'm trying to do it but flet crash in `Packaging Python app...`  step...

I already added to requirements.txt

Flet version: 0.19.0 and 0.20.0

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by frekarls** [comment]
Hi. This is possible, I have built an app with database access through sqlalchemy. I suggest you provide some of your code to see what has gone wrong

**Reply by MoreiraCristiano** [reply]
Did you use the p4a package or just run `flet build apk`?

**Reply by frekarls** [reply]
Sorry... my bad, I only use this in a webapp... haven't tried the build apk...
